math/finding_number_of_digits_in_a_number.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * @author amino19
+ * @file
+ *
+ * \brief Finding number of Digits in a Number
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

Reply:
Ya, already attached a webpage in Explanation part. 
If that needs more changes, pl do acknowledge me!

Reply:
It needs to be added here, too. It's needed for the Doxygen documentation.